### PR TITLE
benchmark: API key and Kafka option arguments

### PR DESCRIPTION
**Pull request by snkas**
Changes to the end-to-end SQL benchmark run script:
- Argument to supply additional Kafka options such as for SSL and SASL configuration
- Argument to supply API key which is included as an HTTP header with each request
- Specify optimized compilation profile
- Comments for configuring pipeline resources

Is this a user-visible change (yes/no): no

### CODE REVIEW

File: benchmark/feldera-sql/README.md [hunk 1]
```diff
@@ -98,5 +98,5 @@ This step has the following sub-steps:
 To run the benchmark itself, run:
 
 ```
-python3 benchmark/feldera-sql/run.py --api-url $FELDERA_API --kafka-broker $KAFKA_FROM_FELDERA
+python3 benchmark/feldera-sql/run.py --api-url $FELDERA_API -O bootstrap.servers=$KAFKA_FROM_FELDERA
```
Comment (blp) [inline benchmark/feldera-sql/README.md:101]: If you're going to delete `--kafka-broker` then you'll need to also update `benchmark/run-nexmark.sh`, which uses it.

**Comment by blp** [comment]
> For the end-to-end benchmark script it adds the following:
> 
>     * Ability to add additional Kafka options such as for SSL and SASL configuration
> 
>     * Supply API key which is included as an HTTP header with each request
> 
>     * Specify optimized compilation profile
> 
>     * Comments for configuring pipeline resources

Thanks for the extra info. Would you mind sticking that into the commit message and force-pushing it? Then it'll be there in the commit log later.

File: benchmark/feldera-sql/README.md (hunk 1 above)
Comment (aehmttw) [reply benchmark/feldera-sql/README.md:101]: Might also want to update https://github.com/feldera/feldera/blob/main/scripts/bench.bash (which runs this script too)

**Review by blp — COMMENTED** [review]
Thank you!

**Pull request merged**